.github/workflows/ci.yml

```diff
@@ -38,16 +38,23 @@ jobs:
         name: Run Terraform validate check
         run: terraform validate
 
+      - id:  install_latest_trivy_version
+        uses: aquasecurity/setup-trivy@v0.2.0
+        with:
+            cache: true
+            version: v0.58.1
+
       - id: run_trivy_config
         name: Run Trivy vulnerability scanner
-        # aquasecurity/trivy-action@v0.16.0
-        uses: aquasecurity/trivy-action@91713af97dc80187565512baba96e4364e983601
+        uses: aquasecurity/trivy-action@master
         with:
           scan-type: config
           trivy-config: trivy.yaml
+          ignore-unfixed: true
           hide-progress: false
           format: sarif
           output: trivy.sarif
+        continue-on-error: true # Allows the job to continue even if this step fails
```

Comment:
Add a TODO to update this when the fix is available in trivy-action, also `@master` is not a good practice